src/Components/MapModal.js

```diff
@@ -63,14 +63,16 @@ export default class ExampleApp extends React.Component {
     /* Sort programResults, largest percentage first */
     sortProgramResults() {
         this.props.programResults.sort((a, b) => {
-            return a.programPercentage < b.programPercentage;
+            return b.programPercentage - a.programPercentage;
```

Comment:
I'm assuming this is an attempt to add sorting - don't think this is fully working yet, unless I'm mistaken!